src/main/java/org/folio/circulationbff/service/UserService.java

```diff
@@ -0,0 +1,8 @@
+package org.folio.circulationbff.service;
+
+import org.folio.circulationbff.domain.dto.UserCollection;
+
+public interface UserService {
+
```

Comment:
Let's either have empty lines both sides of this method or not have them at all

Reply:
done